src/React.rei

```diff
@@ -519,20 +560,25 @@ module Uncurried: {
     "useCallback";
 };
 
-type transitionConfig = {timeoutMs: int};
-
 [@mel.module "react"]
-external useTransition:
-  (~config: transitionConfig=?, unit) =>
-  (callback(callback(unit, unit), unit), bool) =
+external useTransition: unit => (bool, callback(callback(unit, unit), unit)) =
   "useTransition";
 
+module Experimental: {
```

Comment:
should we also include SuspenseList in this module?

Reply:
SuspenseList got unstable by React in https://github.com/facebook/react/pull/27061 and https://github.com/facebook/react/pull/27056#discussion_r1253282784

I'm fine keeping it but it's very useless